src/NuGet.Core/NuGet.ProjectModel/LockFile/LockFileUtilities.cs

```diff
@@ -26,5 +27,20 @@ public static LockFile GetLockFile(string lockFilePath, Common.ILogger logger)
 
             return lockFile;
         }
+
+        public static LockFile SafeGetLockFile(string lockFilePath, Common.ILogger logger)
```

Comment:
Yeah I'm aware of that :D You said, you see how it minimizes the risk, that's what we're agreeing on :D